_includes/configuration/authentication.html

```diff
@@ -8,8 +8,8 @@ <h2>Authentication <small><span class="label label-info">Namespace</span> <span
 
 <p>
 Collins comes bundled with a small in memory authentication mechanism for testing, file based
-authentication, and LDAP based authentication. There is not yet support for LDAPS authentication
-so if you use LDAP we recommend running a slave on the collins host.
+authentication, and LDAP based authentication. We do recommend running a slave on the collins host
```

Comment:
something short about when you might want `file based` vs `LDAP based` authentication would be nice. e.g. `file based (for small or simple installation), and LDAP based authentication (for large or enterprise installation)`

Also "a slave" -> "a ldap slave" would improve readability.